managers/templates/managers/partials/display_projects.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n %}
+{% load static %}
+
+<div class="filtered-projects">
+    <a class="toggled-projects" data-toggle="collapse" href="#multiCollapse-{{ id }}" role="button" aria-expanded="{% if id == "active" %}true{% else %}false{% endif %}" aria-controls="multiCollapse-{{ id }}">
+        {{ header }}
+    </a>
+</div>
+
+<div class="row main-row">
+    <div class="collapse multi-collapse {% if id == "active" %}collapse in{% endif %}" id="multiCollapse-{{ id }}">
+        <div class="card card-body">
+            {% for project in filtered_list %}
+                <div class="col-sm-6 col-md-4 col-xl-4 box-wrapper">
+                    <a href="{% url 'custom-project-detail' pk=project.id %}">
+                        <div class="row project-box">
+                            <div class="row project">
+                                <div class="project-reports">
+                                    <img src="{% static 'users/images/grey/ico-05.png' %}">
+                                </div>
+                                <div class="project-header">
+                                    <h5><b>{{ project.name }}</b></h5>
+                                    <p>
+                                        <span class="glyphicon glyphicon-expand"></span> {{ project.start_date }}
+                                    </p>
+                                </div>
+                            </div>
+                            <div class="row">
+                                {% for user in project.managers.all %}
+                                    <p class="project-managers">
+                                    <span class="glyphicon glyphicon-user"></span> {{ user.first_name }} {{ user.last_name }}
+                                    </p>
+                                {% endfor %}
+                            </div>
+                            <div class="row">
+                                <p>
+                                    <span class="glyphicon glyphicon-user"></span> {{ project.members.count }}
+                                </p>
+                            </div>
+                        </div>
+                    </a>
+                    <div class="row images">
+                        <a href="{% url 'project-report-list' pk=project.pk year=year_for_urls month=month_for_urls %}" title="{% trans 'Check project reports' %}">
+                            <img src="{% static 'managers/images/ico-16.png' %}">
+                        </a>
+                        <a href="{% url 'project-task-activities' pk=project.pk %}" title="{% trans 'Check project task activities' %}">
+                            <img src="{% static 'managers/images/ico-19.png' %}">
```

Comment:
`ico-19.png` is not good icon for this purpose better would be https://fontawesome.com/icons/cog?style=solid

Reply:
I have also changed `ico-16` because it was img and it was very difficult to make it in same size. I used fontawesome icon too